subprojects/reporting/src/main/java/org/gradle/api/reporting/ReportSpec.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.reporting;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.Named;
+
+/**
+ * Common parent for aggregation report types:
```

Comment:
I think this comment should just describe what an aggregration report (this type) is, and let the javadoc tool (or IDE navigation) worry about linking to the known subtypes.

Reply:
Good point, I'll remove them